core/formencoder.go

```diff
@@ -1,12 +1,20 @@
 package core
 
-import "github.com/ggicci/httpin/internal"
+import (
+	"github.com/ggicci/httpin/internal"
+	"strings"
+)
 
 type FormEncoder struct {
 	Setter func(key string, value []string) // form value setter
 }
 
 func (e *FormEncoder) Execute(rtm *DirectiveRuntime) error {
+	tag := rtm.Resolver.Field.Tag.Get("in")
+	if rtm.Value.IsZero() && strings.Contains(tag, "omitempty") {
```

Comment:
This test is error prone. Either check

1. `rtm.Directive.Argv` for `omitempty` defined in a directive, or
2. `rtm.Resolver.GetDirective` for `omitempty` defined as a single directive

Reply:
Thank you! Have gone with option 2.